haiku-games/bevexed/bevexed-20141224.recipe

```diff
@@ -1,14 +1,12 @@
 SUMMARY="A maddeningly-addictive puzzle game"
-DESCRIPTION="
-A maddeningly-addictive puzzle game. The concept is simple: put the tiles in \
-the grid on the left such that the numbers match wherever two tiles touch.
-"
+DESCRIPTION="A maddeningly-addictive puzzle game. The concept is simple: put the tiles in \the grid on the left such that the numbers match wherever two tiles touch."
 HOMEPAGE="https://github.com/HaikuArchives/BeVexed"
-SOURCE_URI="$(HOMEPAGE)/archive/01613e90e6ce3c2b47a592efe8e248db3445f536.tar.gz"
-REVISION="2"
-
-LICENSE="MIT"
 COPYRIGHT="2006 DarkWyrm"
+LICENSE="MIT"
+REVISION="2"
+SOURCE_URI="https://github.com/HaikuArchives/bevexed/archive/01613e90e6ce3c2b47a592efe8e248db3445f536.tar.gz"
+CHECKSUM_SHA256="02665da2df862c05c2d46b80f50834500ad2143761d86da7c6aad97c19c49bd7"
+SOURCE_DIR="BeVexed-01613e90e6ce3c2b47a592efe8e248db3445f536"
```

Comment:
Same remark as before. Using srcGitRev is nice because it allows to specify the commit id only once.
```
srcGitRev="01613e90e6ce3c2b47a592efe8e248db3445f536"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="02665da2df862c05c2d46b80f50834500ad2143761d86da7c6aad97c19c49bd7"
SOURCE_DIR="BeVexed-$srcGitRev"
```